### ISSUE TITLE
Fix alignment issue

### ISSUE THREAD

**Issue by roedoejet** [ReadAlongs/Studio-Web-archived#1]

**Comment by dhdaines** [comment]
Hi Aidan,  I'm a bit confused by the soundswallower.js and soundswallower.wasm in assets/ - it looks like they've been pre-compiled elsewhere and then copied in, and there is some extra code in them.  I'd like to replace them with the package from NPM, but I don't see the source for your extra methods anywhere, could you add it?

**Comment by dhdaines** [comment]
SoundSwallower 0.3.1 on NPM should have a working .d.ts file, BTW.  This seems to work for me (with `--esModuleInterop --downlevelIteration --lib es2015,dom`):

```typescript
import soundswallower_factory from "soundswallower";
import {SoundSwallowerModule, Config} from "soundswallower";

(async () => {
    const soundswallower: SoundSwallowerModule = await soundswallower_factory();
    const config: Config = new soundswallower.Config();
    console.log(config.get("samprate"));
})();

```

**Comment by dhdaines** [comment]
Now fixed with SoundSwallower 0.3.2 in #7 - the actual bug was fixed in https://github.com/ReadAlongs/SoundSwallower/commit/6c6cdb4d05e814ad63f076e5833c6a9553005ca4
